src/user/SignUpForm.js

```diff
@@ -0,0 +1,80 @@
+// Sign Up Form allows anyone to sign up for an account to access their personal dashboards
+// The contact form requests First Name, Last Name, Email, Password & Confirm Password.
+// The sign up form then redirects the user to their personal dashboards
+
+import React from 'react';
```

Comment:
Instead of creating a React.Component below, let's keep it all in the import line.
```suggestion
import React, {Component} from 'react';
```

And then down below
```
class SignUpForm extends Component {
```

Reply:
No worries! Fixed up! :)